src/components/Pickers/Suggestion/Suggestion.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../common/common';
+
+.ms-SearchMore-Button {
+  @include button-reset();
+  position: relative;
+  height: auto;
+  width: 100%;
```

Comment:
Hmmm, the button doesn't seem to work that way. without width 100% it doesn't fill the space given and looks odd. I'll research.